docs/docs/concepts/dialogue-understanding.mdx

```diff
@@ -21,9 +25,62 @@ In the CALM approach, _Dialogue Understanding_ aims to represent how the end use
 wants to progress the conversation.
 There is currently only one command generator available: the `LLMCommandGenerator`.
 
-## Using the LLMCommandGenerator in Your Bot
+Like its name indicates, the `CommandGenerator` generates "commands" that are then internally processed to
+trigger operations on the current conversation. Here is a reference of all supported commands:
+
+### StartFlowCommand
+
+This command indicates the assistant should start a new [flow](./flows.mdx).
+
+### CancelFlowCommand
+
+This command indicates the assistant should cancel the current [flow](./flows.mdx).
+It powers [Conversation Repair's Cancellation use case](./conversation-repair.mdx#3-cancellations).
+
+### SetSlotCommand
+
+This command indicates the assistant should set a [slot](./domain.mdx#slots) to a given value.
+
+### CorrectSlotsCommand
+
+This command indicates the assistant should change the value of a given [slot](./domain.mdx#slots)
+to a new value. It powers [Conversation Repair's Correction use case](./conversation-repair.mdx#2-corrections).
+
+### ClarifyCommand
```

Comment:
@dakshvar22 we discussed possibly moving this to pro. What's your take?

Reply:
@amn41 if the product decision is not made yet, I would push for not blocking this PR and instead update the docs when code is actually moved to pro. Are you ok with that? (same for the question below)

Reply:
Discussing this [here](https://rasa-hq.slack.com/archives/C047U7ZBKM1/p1695735512041479). Happy to create a follow-up ticket which refactors the docs for it once we have a decision to not block this ticket.

Reply:
Follow up ticket - https://rasahq.atlassian.net/browse/ENG-578